dashboard/src/shared/App.test.ts

```diff
@@ -93,9 +93,15 @@ describe("App", () => {
     });
     it("throws an error if returns an error 404", async () => {
       moxios.stubRequest(/.*/, { status: 404 });
-      expect(App.delete("foo", "default", false)).rejects.toThrow(
-        "Request failed with status code 404",
-      );
+      let errored = false;
```

Comment:
this change is not related to the PR, I am not sure why when I was running test, this was failing so I had to rewrite it.